roles/tls_signing/meta/argument_specs.yml

```diff
@@ -0,0 +1,63 @@
+# Copyright 2024 Cloudera, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+
+argument_specs:
+  main:
+    short_description: "Sign of CSRs by a CA Server"
+    description:
+      - Generates a signed TLS certificate from a specified list of Certificate Signing Requests (CSRs).
+      - The role will optionally override and backup existing certs of the same name if required.
+      - Upon completion the signed certs are copied back to the Ansible controller.
+    author:
+      - "Jim Enright <jenright@cloudera.com>"
+    options:
+      csrs_to_sign:
+        description: "List of full path locations of the CSRs to sign."
+        type: "list"
+        elements: "str"
+        required: true
+      copy_from_controller:
+        description: "Flag to specify if the CSRs should be copied from the Ansible controller."
+        type: "bool"
+        default: true
+      override_old_certs:
```

Comment:
This is a confusing parameter. If set to `false`, then the role _will_ override, but if set to `true`, the role will back up the cert. The functionality is right; the naming is not.

Reply:
This has simplified with the use of [community.crypto.x509_certificate](https://docs.ansible.com/ansible/latest/collections/community/crypto/x509_certificate_module.html). I've renamed the parameter to `backup_old_certs` and use the parameter in the `backup` argument to community.crypto.x509_certificate.